influxdb3_catalog/src/catalog.rs

```diff
@@ -948,8 +948,8 @@ impl TableDefinition {
         self.column_map.get_by_right(&name.into()).copied()
     }
 
-    pub fn column_id_to_name(&self, id: ColumnId) -> Option<Arc<str>> {
-        self.column_map.get_by_left(&id).cloned()
+    pub fn column_id_to_name(&self, id: &ColumnId) -> Option<Arc<str>> {
```

Comment:
👍 We should probably do this to all the `get_by_id` methods in the catalog.

Reply:
Good idea - pushed it in [commit](https://github.com/influxdata/influxdb/pull/25521/commits/c6edccbad8e3eba7f4b945b6a2dfbba952117bd8)

Reply:
I'll merge this now @hiltontj - if I've missed any methods, we could pick it up in subsequent PRs.